package.json

```diff
@@ -4,6 +4,7 @@
     "description": "",
     "main": "dist/index.js",
     "scripts": {
+        "dev": "NODE_ENV=development rollup --config --watch",
```

Comment:
What about `rollup --config --watch NODE_ENV=development` I think it's a better practice to put the command itself first, so it's immediately clear what it does, and after all the configuration details, flags, etc?

Reply:
> What about `rollup --config --watch NODE_ENV=development` I think it's a better practice to put the command itself first, so it's immediately clear what it does, and after all the configuration details, flags, etc?

hm, did not think from this perspective. Agree

Reply:
just tested, it is not working. `NODE_ENV` has to be before a command

Reply:
Oh, yes, that's right, maybe there should be something like webpack has, e.g. `--mode` flag. I'll check

Reply:
@fidelman I think that't that we need https://rollupjs.org/guide/en/#--environment-values